### ISSUE TITLE
add a separate BGR reader 

### ISSUE THREAD

**Issue by heeler** [AllenCellModeling/aicspylibczi#36]
Currently, BGR images are split into their base grayscale type (BGR24 => 3 x Gray8 = 3 x uint8)

I have received requests to return the interleaved data as opposed to splitting them apart. 
I will also try to apply any color scaling Zen wrote into the file so it has the expected appearance.

**Comment by MosGeo** [comment]
I am getting an error (below) when reading a specific BGR24 czi file with multiple channels, or layers (each channel is itself a colored image). Is this related to the above? I am still invistigating as I just started using this package.

```
PylibCZI_ImageSplitChannelException       Traceback (most recent call last)
 in 
----> 1 im = czi.read_image()

C:\Python38\lib\site-packages\aicspylibczi\CziFile.py in read_image(self, **kwargs)
    313         m_index = self._get_m_index_from_kwargs(kwargs)
    314 
--> 315         image, shape = self.reader.read_selected(plane_constraints, m_index)
    316         return image, shape
    317 

PylibCZI_ImageSplitChannelException: ImageSplitChannelExcetion: attempting to split channels Channel should be zero or unset but has a value of 1 not sure how to procede in assigning channels.
```

**Comment by heeler** [comment]
I'm wondering if you could share the output of `dims_shape()` for the specific file. Also is there any chance you'd be willing to share the file with me? We tend to acquire each channel as a Gray16/uint16 and the meaning of a channel is fairly clear. When I had to unpack the BGR images it was unclear to me what it would mean to have a BGR image with channels. If you could explain the structure to me and Ideally provide a file to test against.  I can probably address this fairly quickly. Thank you for taking the time to post here.

**Comment by MosGeo** [comment]
Thanks for the prompt reply. Please check your email for a test file.
The shape is:    [{'X': (0, 147), 'Y': (0, 81), 'C': (0, 7), 'S': (0, 1)}]
The pixel type: BGR24

Note that I am able to load the first channel perfectly by setting C=0 in `read_image`. The resulatant shape from the read operation is: [('B', 1), ('S', 1), ('C', 3), ('Y', 81), ('X', 147)]. Now the channels are the actual RGB data (or BGR) I think I need to swap them to display it correctly. 

In conclusion, I think if you just remove the the if statement, it should work for other channels (C=1,C=2, ...)

**Comment by heeler** [comment]
I have made a branch that will allow the user to pull out any requested Channel as a ndarray. 
Your comment above is correct. I'm being a little cautious as this is the first example of BGR w/ channels that I've seen. It should be under PR soon and then released. 

Thanks for sharing the data file and for the suggestions. Would a reader function that returned BGR in a more native interlaced format be desirable?  [Y, X, BGR{0,1, 2}]

**Comment by heeler** [comment]
I think this is addressed by some of the recent changes, 2.7.1 . I ended up not making a BGR specific function. What should happen now is that if you have a 7 channel BRG image and you pull access it as a BGR it will provide 21 channels back, 3*7. If this isn't workable for you please reopen this issue and I'll take another look. Thanks.
